marklogic-data-hub-central/ui/src/components/modeling/property-table/property-table.tsx

```diff
@@ -321,6 +321,7 @@ const PropertyTable: React.FC<Props> = (props) => {
       let externalEntity = modelingOptions.entityTypeNamesArray.find(entity => entity.name === parseType[1])
       return {
         datatype: 'array',
+        facet: facetable,
```

Comment:
Let's use the term "facetable" in the UI as well.

Reply:
"facet" is referenced a lot in the UI already, but go ahead and change it @saarimrahman

Reply:
I think it's important to distinguish between the two concepts, which are - 1) the thing that appears in search results with values and counts, and 2) whether or not a property should have a thing appear in search results for it. Thus, two separate terms are warranted.